modules/gsp-cluster/grafana.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_iam_role" "grafana" {
+  name               = "${var.cluster_name}-grafana"
+  description        = "Role the Grafana process assumes"
+  assume_role_policy = "${data.aws_iam_policy_document.trust_kiam_server.json}"
+}
+
+resource "aws_iam_role_policy" "grafana" {
+  name =  "${var.cluster_name}-grafana"
+  role = "${aws_iam_role.grafana.id}"
+
+  policy = <<EOF
```

Comment:
Can we use aws_iam_policy_document etc. like modules/gsp-user/iam.tf does?

Reply:
Done.